dobson/utils.py

```diff
@@ -0,0 +1,62 @@
+import configparser
+import json
+import os
+import subprocess
+from collections import namedtuple
+
+
+Device = namedtuple('Device', ['presence', 'user', 'model'])
+
+config = configparser.ConfigParser()
+config.read('dobson/config.ini')
+
+
+SLACK_API_TOKEN = config['dobson']['SlackApiToken']
+
+# Only allow querying for presence in certain channels
+ALLOWED_CHANNEL_IDS = json.loads(config['dobson']['AllowedChannelIds'])
+
+# The user ID of the dobson bot
+DOBSON_SLACK_ID = config['dobson']['DobsonSlackId']
+
+# file to log current MAC Addresses in
+MAC_LOG_FILE = config['dobson']['MacAddressLogFile']
+
+# create KNOWN_DEVICES, which maps mac addresses (str) to Device instances
+if not os.path.isfile(config['dobson']['KnownDevicesFile']):
+    with open(config['dobson']['KnownDevicesFile'], 'w') as f:
+        f.write('{}')
+    KNOWN_DEVICES = {}
+else:
+    with open(config['dobson']['KnownDevicesFile'], 'r') as f:
+        KNOWN_DEVICES = json.loads(f.read())
+    for key, val in list(KNOWN_DEVICES.items()):
+        KNOWN_DEVICES[key] = Device(**val)
+
+
+def get_devices():
+    """Get the list of MAC addresses of connected devices"""
```

Comment:
Given this method has been split, I think the docstring isn't especially accurate now, maybe something like `Get all recognized connected devices` or something would be better?

Reply:
Sounds good.